proxy/grpcproxy/watcher_single.go

```diff
@@ -53,9 +52,8 @@ func (ws watcherSingle) run() {
 	defer close(ws.donec)
 
 	for wr := range ws.ch {
-		ws.rev = wr.Header.Revision
+		ws.lastSeenRev = wr.Header.Revision
```

Comment:
we use w.rev not lastSeenRev for coalescing watchers. see https://github.com/coreos/etcd/pull/5984/files#diff-044db55c97da02fe244f940eba0c0576R72.